models/hl7-v3/prescription.ts

```diff
@@ -248,15 +248,19 @@ export class PrescriptionPredecessor implements ElementCompact {
 }
 
 export abstract class PrescriptionAnnotation implements ElementCompact {
-  _attributes: core.AttributeClassCode & core.AttributeMoodCode = {
+  _attributes: core.AttributeClassCode & core.AttributeMoodCode & core.AttributeNegationInd = {
     classCode: "OBS",
-    moodCode: "EVN"
+    moodCode: "EVN",
+    negationInd: undefined
   }
 
   code: codes.PrescriptionAnnotationCode
 
-  constructor(code: codes.PrescriptionAnnotationCode) {
+  constructor(code: codes.PrescriptionAnnotationCode, negationInd?: boolean) {
     this.code = code
+    if (negationInd !== null && negationInd !== undefined) {
```

Comment:
shame this looks dumb, maybe it would be nicer just
```
if (negationInd === true) this._attributes.negationInd = "true"
if (negationInd === false) this._attributes.negationInd = "false"
```
or curly braces